content/en/continuous_integration/guides/identify_highest_impact_jobs_with_critical_path.md

```diff
@@ -0,0 +1,83 @@
+
+Filtering and Exclusive Time computation of CI Jobs on the Critical Path is available for the following CI Providers:
+* [GitLab][3]
+
+You can still [highlight which CI Jobs are on the critical path][4] using the Pipeline Execution detail view across all CI providers.
+
+## Identify the CI Jobs to improve your CI Pipeline
+
+### Using the facet
+
+You can use the facet `@ci.on_critical_path` or `@ci.critical_path.exclusive_time` to identify which CI Jobs are on the critical path in your CI Pipelines. Using those facets, you can create your custom dashboards and notebooks for your needs.
+
+{{< img src="continuous_integration/critical_path_facets.png" alt="Filter using critical path facets" width="90%">}}
+
+Notice that these facets are only available using the `ci_level:job` in your queries.
+
+### Using the dashboard template
+
+You can also import the [CI Visibility - Critical Path][1] dashboard template:
+- Open [civisibility-critical-path-gitlab-dashboard.json][1] dashboard template and copy the content in the clipboard.
+- Create a [New Dashboard][2] in Datadog.
+- Paste the copied content in the new dashboard.
+- Save the dashboard.
+
+{{< img src="continuous_integration/critical_path_dashboard.png" alt="Critical path dashboard for CI Visibility" width="90%">}}
+
+#### Terminology
+
+| Column                                | Description                                                                                                                                                      |
+|---------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Total Exclusive Time On Critical Path | Sum all exclusive time of the job. It gives an approximation of the potential saved time the involved pipelines might have.                                      |
+| Avg Exclusive Time On Critical Path   | Avg exclusive time of a particular job on the critical path. This measures the potential reduction of a pipeline duration if the job reduces its exclusive time. |
+| Rate On Critical Path                 | Measures the porcentage that a job is on the critical path compared when that job is not.                                                                        |
+
+##### Example
+
+In the previous image, we can observe that a CI job called `metrics` is a potential candidate for improvement, as its total exclusive time is the highest. The average exclusive time is around 21 minutes, meaning there is room for improvement of up to 21 minutes for this CI job. 
+
+Since we know this CI job is on the critical path, any improvements made to it will be reflected in the corresponding CI pipelines, reducing a similar amount of time.
```

Comment:
```suggestion
Since we know this CI job is on the critical path 43.5% of the time, we can hope to reduce the average pipeline duration by 21 minutes for 43.5% of the pipeline executions.
```